rest/endpoints/user_recordings/profile_recordings.yml

```diff
@@ -32,12 +32,64 @@ paths:
   /user_profiles/{user_profile_id}/recordings:
     get:
       description: |
-        * **Available since:** NOT YET AVAILABLE
+        * **Available since:** https://jira.aminocom.com/browse/BPLAT-12427
       summary: Search for recordings of a given user profile.
       tags:
         - Recordings
       operationId: searchRecordedSlots
-      # TODO
+      parameters:
+        - in: path
+          name: user_profile_id
+          required: true
+          description: The identity of the user profile to fetch recording groups for.
+          schema:
+            type: string
+        - in: query
+          name: page_number
+          required: false
+          description: Page number if the results have more items than ipp. Default = 1
+          schema:
+            type: integer
+        - in: query
+          name: language
+          description: >-
+            The PK of the language to use, which will be a 3 character ISO 639-2 string (e.g.: fin).
+            If this is omitted the system defined default will be used. If the requested language is not available
+            then the system will return translations in order of the system defined preferences
+          required: false
+          schema:
+            type: string
+        - in: query
+          name: filter
+          description: >-
+            Please see http://jsonapi.org/recommendations/#filtering. The following are attributes to filter on: [search]
+            Supports search by title keywords, when no search string provided, API returns all the recorded slots for the requested user_profile_id
+          required: false
+          schema:
+            type: string
+      responses:
+        200:
+          description: "Successful response on updated resource"
+          content:
+            application/json:
+              schema:
+                type: "object"
+                required:
+                  - data
+                properties:
+                  data:
+                    $ref: "https://raw.githubusercontent.com/booxmedialtd/api-documentation/master/rest/snippets/UserRecordings/RecordingEntity.yml#/RecordingEntity"
```

Comment:
It should be an array of RecordingEntities. Example: https://github.com/booxmedialtd/api-documentation/blob/812e84c3bd3e6b27fd93551a646a593bddf1c303/rest/endpoints/user_recordings/profile_recordings.yml#L409-L414

preview: https://confluence.aminocom.com/display/BPLAT/%5BWIP+-+Garrick%5D+API+draft#/Recordings/searchRecordedSlots